### PR TITLE
Removal of all references to --contig-dir option

### PR DESCRIPTION
**Pull request by csieffert**
Removal of --contig-dir option from the code.  Resolves issue #18.

### CODE REVIEW

**Comment by apetkau** [comment]
This looks great.  Could you also remove any mention of `contig-dir` and `contig_fastq` within `README.md`?  Also, while we're at it, could you remove mentiones of `contig1.png` in the README.md file, listing how we generate images using GView?  Thanks.

**Comment by apetkau** [comment]
Also, could you remove the stages dealing with MUMMer, which relate to the "contig-dir"?  https://github.com/apetkau/core-phylogenomics/blob/master/lib/Pipeline/Mapping.pm#L225

**Comment by csieffert** [comment]
All of the additional contig-dir references and all files/references associated with mummer.  Let me know if you find any that I missed.  Thanks!

File: etc/pipeline.conf [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
```
Comment (apetkau) [inline etc/pipeline.conf]: I believe this file was accidentally committed, could you remove it?

File: lib/JobProperties.pm [hunk 2]
```diff
@@ -193,7 +192,6 @@ sub _set_defaults
 	$self->set_file('delta-filter', $delta_filter) if ((defined $delta_filter) and (-e $delta_filter));
```
Comment (apetkau) [inline lib/JobProperties.pm]: Could you remove the **delta-filter,show-aligns,show-snps** here as well?

**Comment by apetkau** [comment]
Awesome.  I believe you missed the line **52: Alternatively, in addition to the above files ...** and **73: For any assembled contigs ...** in the `README.md` file, but beyond that it looks great.  Thanks for doing this.  I'll also have @Takadonet to make sure we didn't remove any code he still wanted in there.

**Comment by csieffert** [comment]
Lines removed from README.md in commit 6701c2b0f4b50c63600852d4a0ac6c0eaf65e39a.  Thanks

File: lib/JobProperties.pm [hunk 3]
```diff
@@ -190,11 +186,6 @@ sub _set_defaults
 	$self->set_file('shuf', $shuf) if ((defined $shuf) and (-e $shuf));
 	
 	$self->set_file('nucmer', $nucmer) if ((defined $nucmer) and (-e $nucmer));
```
Comment (apetkau) [inline lib/JobProperties.pm]: I think this is the last line that should be removed, since we don't use nucmer anymore.

**Comment by csieffert** [comment]
Line removed from JobProperties.pm in commit 0b9e39151deb744ee8bc3c0c41903bfe4334b4de.

File: lib/JobProperties.pm (hunk 3 above)
Comment (csieffert) [inline lib/JobProperties.pm]: Line removed from JobProperties.pm in commit  b9e39151deb744ee8bc3c0c41903bfe4334b4de.

**Comment by apetkau** [comment]
This looks great.  Thanks.  Merging.

**Pull request merged**